### ISSUE TITLE
verilator doesn't operate clk24

### ISSUE THREAD

**Issue by mattvenn** [Spritetm/hadbadge2019_fpgasoc#139]
I'm looking at integrating new bits into the SOC, so I want to check they work.
My design uses the 24 and 48 mhz clock, but only the 48 and 96 mhz clock is supplied by verilator.
Is there a reason for this?

**Comment by smunaut** [comment]
Yeah, the 24 MHz clock wasn't used and each new clock increases sim time so we left it out so far.

**Comment by mattvenn** [comment]
So for example, in your fork that includes the HUB75 core, if I want to test that in simulation I will have to add the 24mhz clock to verilator (and fix those parameter issues)
